features/machine/machine_healthcheck.feature

```diff
@@ -130,3 +130,41 @@ Feature: MachineHealthCheck Test Scenarios
       | mhc-<%= machine_set.name %>-1: total targets: 1,  maxUnhealthy: 0, unhealthy: 1. Short-circuiting remediation   |
       | mhc-<%= machine_set.name %>-2: total targets: 1,  maxUnhealthy: 90%, unhealthy: 1. Short-circuiting remediation |
     """
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-28718
+  @admin
+  @destructive
+  Scenario: [MHC] - Machine Node startup timeout should be configurable
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Then I use the "openshift-machine-api" project
+    And admin ensures machine number is restored after scenario
+
```

Comment:
@miyadav small nitpick... extra empty line 143 & 145

Reply:
thanks @pruan-rht  , will take care of these going forward.